fields/default/fields_list.json

```diff
@@ -3174,7 +3174,7 @@
         "path": "ads.history.items[].photos.grz",
         "description": "История объявлений: Список ГРЗ c URI фотографии, на которых они распознаны",
         "types": [
-            "array"
+            "object"
```

Comment:
@tarampampam Found an error. This field wass added in `v3.2.0` in [this PR](https://github.com/avtocod/specs/pull/48/files). Actually this field contains object, not array

Reply:
Nice job!

Reply:
Do not forget to update your changes in changelog file